gomint-server/src/main/java/io/gomint/server/entity/EntityPlayer.java

```diff
@@ -1544,8 +1545,15 @@ public void firstSpawn() {
 
         // Now its time for the join event since the player is fully loaded
         PlayerJoinEvent event = this.getConnection().getNetworkManager().getServer().getPluginManager().callEvent( new PlayerJoinEvent( this ) );
+        event.setJoinMessage( ChatColor.YELLOW + this.getDisplayName() + ChatColor.YELLOW + " joined the game." );
```

Comment:
The way you do this overrides any join message set by plugins. I suggest adding a parameter to PlayerJoinEvent constructor for the default message instead.

Reply:
Oh i thought that but i wasnt sure